README.md

```diff
@@ -573,6 +573,13 @@ const proxy = createProxyMiddleware({
 
 Check out [interception recipes](https://github.com/chimurai/http-proxy-middleware/blob/master/recipes/response-interceptor.md#readme) for more examples.
 
+## Issues over IPv6 and localhost
+From Node 17+: `For dns lookups, Node.js no longer prefers IPv4 over IPV6`.
+That said, `localhost` is not a valid hostname to proxy to, since it's only exists on IPv4.
+It will probably affect your development environment (if not already), and the easiet ways to avoid this problem are:
+1. Replace all proxy to `localhost` into proxy to `127.0.0.1`
+2. (Not recommended) Add this flag when running node on locally: `node index.js --dns-result-order=ipv4first`
```

Comment:
This does not match my understanding of how it works: https://github.com/nodejs/node/issues/40702#issuecomment-1103623246

Reply:
@lydell rephrased.

Reply:
@chimurai updated.